docs/conf.py

```diff
@@ -53,9 +53,9 @@ def get_version():
 exclude_patterns = ['_build']
 default_role = 'obj'
 intersphinx_mapping = {
-    'python': ('http://python.readthedocs.io/en/latest/', None),
-    'django': ('http://django.readthedocs.io/en/1.9.x/', None),
-    'sphinx': ('http://sphinx.readthedocs.io/en/latest/', None),
+    'python': ('https://python.readthedocs.io/en/latest/', None),
+    'django': ('https://django.readthedocs.io/en/1.9.x/', None),
```

Comment:
This should probably be 1.11.x which is the version that we are using.

Reply:
Happy to change it as part of this PR, I was just fixing the URL scheme.  Just let me know